consensus/src/aggregation/engine.rs

```diff
@@ -242,10 +249,10 @@ impl<
             buffer_pool: self.journal_buffer_pool.clone(),
             write_buffer: self.journal_write_buffer,
         };
-        let journal = Journal::init(self.context.with_label("journal"), journal_cfg)
+        let journal = Journal::init(context.clone().with_label("journal"), journal_cfg)
```

Comment:
with_label() already clones. We should grep for any `.clone().with_label` and collapse to `.with_label`